core/src/main/tut/README.md

```diff
@@ -55,21 +87,41 @@ the [config files documentation](docs/config-files.html):
 }
 ```
 
-Then, load the configuration ([in this case from the classpath](docs/config-files.html)):
+Finally, load the configuration:
 
 ```scala
 loadConfig[MyClass](conf)
```

Comment:
Stupid question: Why does this work? I don't see a `conf` identifier defined anywhere in this file.

Reply:
The response is also stupid; that works because, unfortunately, we aren't running that by tut (note the `scala` block in the markdown) :(

#320 updated some aspects of our documentation and one of the changes introduced was to show an example of an `application.conf` instead of showing a `val conf = ConfigFactory.parseString(...)` call, since the latter is not at all a common use case for pureconfig. Apparently [my concerns](https://github.com/pureconfig/pureconfig/pull/320#discussion_r151871712) with not running this with tut were shown to be justified much sooner than I would like to... 

On the next PR I'll make this run with tut, even if I have to add some sample `application.conf` to the resources in order for the code to compile. Right now, I'll leave this as it is since the "bug" wasn't introduced with the changes on this PR.

Reply:
Totally OK. Thanks for taking a look. Thanks again for your efforts.